lib/travis/api/v3/queries/log.rb

```diff
@@ -58,8 +58,8 @@ def hostname(name)
       "#{name}#{'-staging' if Travis.env == 'staging'}.#{Travis.config.host.split('.')[-2, 2].join('.')}"
     end
 
-    def logs_model
-      return Travis::API::V3::Models::RemoteLog if Travis.config.logs_api.enabled?
+    private def logs_model
```

Comment:
this is in the private section (see above), you don't need the indicator on the method again.